book/chapter-04.md

```diff
@@ -504,25 +506,28 @@ In the context of the expectation and outcome framework of the main chapter, the
+As noted above, the original delta rule was published by [@WidrowHoff60], followed by the critique by [@MinskyPapert69], showing that such models could not learn a large class of basic but nonlinear logical functions, for example the XOR function.   XOR states that the output should be true (active) if *either* one of two inputs are true, *but not both*.  This requires a strong form of nonlinearity that simply could not be represented by such models.  In retrospect, it should have been obvious that the problem was the use of a two-layer network, but as often happens, this critique left a bad "odor" over the field, and people simply pursued other approaches (mainly symbolic AI, which Minsky was an advocate for).
 
-Then, roughly 26 years later, David Rumelhart and colleagues published a paper on the backpropagation learning algorithm, which extended the delta-rule style error-driven learning to networks with three or more layers. The addition of the extra layer(s) now allows such networks to solve XOR and any other kind of problem (there are proofs about the universality of the learning procedure). The problem is that above, we only considered how to change weights from a sending unit *y* to an output unit *z*, based on the error between the target *t* and actual output activity. But for multiple stages of hidden layers, how do we adjust the weights from the inputs to the hidden units?  Interestingly, the mathematics of this involves simply adding a few more steps to the chain rule. 
+Then, roughly 26 years later, David Rumelhart and colleagues published a paper on the backpropagation learning algorithm, which extended the delta-rule style error-driven learning to networks with three or more layers. The addition of the extra layer(s) now allows such networks to solve XOR and any other kind of problem (there are [proofs](https://en.wikipedia.org/wiki/Universal_approximation_theorem) about the universality of the learning procedure). The problem is that above, we only considered how to change weights from a sending unit *y* to an output unit *z*, based on the error between the target *t* and actual output activity. But for multiple stages of hidden layers, how do we adjust the weights from the inputs to the hidden units?  Interestingly, the mathematics of this involves simply adding a few more steps to the chain rule. 
 
 The overall derivation is as follows. The goal is to again minimize the error (SSE) as a function of the weights, 
 $$ \Delta w_{ij} = -\frac{\partial SSE}{\partial w_{ij}} $$
+
+TODO: I think we need to define SSE in terms of $ \eta $ before introducing a derivative that uses $ \eta $! Should it be $x$ instead?
```

Comment:
@rcoreilly please take a look at this. It wasn't clear to me where the η came from.